chips/stm32f4xx/src/clocks/mod.rs

```diff
@@ -0,0 +1,991 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
```

Comment:
Please rename this file to match its contents.

Reply:
This file defines a structure called `Clocks` which encapsulates all clocks present on the chip. Do you want me to rename the file to `clocks.rs`? I find this ugly: `crate::clocks::clocks::Clocks`.

Reply:
Yes. My fault. Sorry!

Reply:
Renamed the file.